ui/components/multichain/product-tour-popover/product-tour-popover.stories.js

```diff
@@ -24,7 +24,13 @@ export default {
       control: 'text',
     },
     onClick: {
-      control: 'onClick',
+      action: 'onClick',
+    },
+    onHide: {
+      action: 'onHide',
+    },
+    closeMenu: {
+      action: 'closeMenu',
```

Comment:
Fixes some propType errors in the story

<img width="1439" alt="Screenshot 2023-04-20 at 2 42 15 PM" src="https://user-images.githubusercontent.com/8112138/233269809-663b7894-f0c4-4225-9b02-583314d0fe28.png">